ci-connect-cli/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/keptn-sandbox/keptn-git-toolbox/ci-connect-cli
+
+go 1.16
+
+require (
+	github.com/go-git/go-git/v5 v5.4.2
```

Comment:
Without the patch of @yeahservice (`github.com/yeahservice/go-git/v5 v5.4.2-aws-patch`), using an AWS code commit repository for the keptn git repo will not be possible.